src/client/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -273,6 +273,44 @@ public T WithLegacyCacheCompatibility(bool enableLegacyCacheCompatibility = true
             return (T)this;
         }
 
+        /// <summary>
```

Comment:
This is good, but you might want to pause to until the IIdentityLogger is finalized. If we can achieve dynamic log level changed there, we don't need to modify our existing implementation.